tests/unit/helpers.js

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import _ from 'lodash';
+import ComponentTree from 'react-component-tree';
+
+
+let _container;
+
+// If there's a container on the page named #test-area, use that, otherwise
+// create a detached div and use that. The former case is useful when running
+// the tests in your own browser so you can debug how the component is
+// rendered.
+const testArea = document.querySelector('#test-area');
+
+if (testArea) {
+  _container = testArea;
+} else {
+  _container = document.createElement('div');
+}
+
+
+/**
+ * Render a component into the DOM.
+ *
+ * @param {React class} Component
+ * @param {Object} fixture
+ * @param {DOM} container You should set this to this.container inside your
```

Comment:
Remove this as it's no longer used. I made an [issue](https://github.com/NiGhTTraX/react-test-buffet/issues/5) to update upstream 😄